db/migrations/20230404074457_create_product_image_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE product_image {
```

Comment:
```suggestion
CREATE TABLE products_images {
```
중간테이블 네이밍 컨벤션 관련 수정입니다!
추가로 다대다 관계의 이미지가 있다고 픽스하고 진행하시는 걸로 이해하면 될까요?

Reply:
네 다대다로 생각하고 진행하겠습니다.